meta-freedesktop/recipes-multimedia/webm/libvpx_1.5.0.bb

```diff
@@ -23,6 +23,8 @@ export LD = "${CC}"
 VPXTARGET_armv5te = "armv5te-linux-gcc"
 VPXTARGET_armv6 = "armv6-linux-gcc"
 VPXTARGET_armv7a = "armv7-linux-gcc"
+VPXTARGET_x86-64 = "x86_64-linux-gcc"
+VPXTARGET_i586 = "x86-linux-gcc"
```

Comment:
According to yocto it very much looks like 64bit intel is with a dash, i586 looks correct too.

I am borderline certain that these words come from the machine configs [here](http://git.yoctoproject.org/cgit/cgit.cgi/poky/tree/meta/conf/machine/include/tune-i586.inc) and [here](http://git.yoctoproject.org/cgit/cgit.cgi/poky/tree/meta/conf/machine/include/x86/arch-x86.inc)

That said, Flatpak currently support 32/64bit Intel and Arm architectures. As such I think it would be better to land this patch with a unilateral change for all supported arches right ?

An extra line to fix do this would probably look like:
```
VPXTARGET_aarch64 = "armv8-linux-gcc"
```

Based on the [yocto config](http://git.yoctoproject.org/cgit/cgit.cgi/poky/tree/meta/conf/machine/include/arm/arch-arm64.inc) and the [libvpx readme](https://github.com/webmproject/libvpx/)

Reply:
Looks like it might be `arm64-linux-gcc` according to https://sources.debian.net/src/libvpx/1.6.1-3/debian/rules/#L33.

Reply:
This is what fedora does:
```
%ifarch %{ix86}
%global vpxtarget x86-linux-gcc
%else
%ifarch x86_64
%global vpxtarget x86_64-linux-gcc
%else
%ifarch armv7hl
%global vpxtarget armv7-linux-gcc
%else
%global vpxtarget generic-gnu
```

Reply:
I wonder about the i586 though.
In other places we use x86, for instance:
```
$ git grep _x86
meta-freedesktop/recipes-core/tasks/task-freedesktop-contents-sdk.bb:RDEPENDS_${PN}_append_x86-64 = " valgrind valgrind-dev libasan-staticdev libubsan-staticdev liblsan-staticdev libtsan-staticdev nasm yasm"
meta-freedesktop/recipes-core/tasks/task-freedesktop-contents-sdk.bb:RDEPENDS_${PN}_append_x86 =    " valgrind valgrind-dev libasan-staticdev libubsan-staticdev nasm yasm"
```

And in freedesktop-sdk-build-yocto:
```
case "$architecture" in
    i386) MACHINE="qemux86";;
    x86_64) MACHINE="qemux86-64"; EXTRA='DEFAULTTUNE = "x86-64"';;
    arm) MACHINE="qemuarmv7a";;
    aarch64) MACHINE="qemuarm64";;
    *) echo "Error: Unhandled architecture '$architecture'"; exit 1;;
esac

```